OpenRA.Mods.Common/UtilityCommands/ImportLegacyMapCommand.cs

```diff
@@ -390,7 +390,7 @@ public static void LoadActors(IniFile file, string section, List<string> players
 					if (health != 100)
 						initDict.Add(new HealthInit(health));
 					if (facing != 0)
-						initDict.Add(new FacingInit(facing));
+						initDict.Add(new FacingInit(255 - facing));
```

Comment:
Instead of hard coding this, it would be better to define a configurable default argument to use 8-bit int max here